Source/FikaAmazonAPI/ReportGeneration/ReportManager.cs

```diff
@@ -34,6 +34,27 @@ private string GetFeedbackFromDate(AmazonConnection amazonConnection, DateTime f
         {
             return amazonConnection.Reports.CreateReportAndDownloadFile(ReportTypes.GET_SELLER_FEEDBACK_DATA, fromDate);
         }
+
+        #endregion
+
+        #region Performance
+        public SellerFeedBackDataResult GetSellerFeedBackDataFromDays(int days)
```

Comment:
Method name GetSellerFeedBackDataFromDays but it should be for Performance

Reply:
the GET_SELLER_FEEDBACK_DATA [report](https://github.com/amzn/selling-partner-api-docs/blob/main/references/reports-api/reporttype-values.md#performance-reports)  is in the Performance Report Category

Reply:
yes , but i ready did that report under name (GetFeedbackFromDays) your new name (GetSellerFeedBackDataFromDays)

Reply:
Can you check here only this point

Reply:
how about you suggestion?

Reply:
Can you change to this report (GET_V2_SELLER_PERFORMANCE_REPORT) because feedback already there

Reply:
ok. i will remove this method(GetSellerFeedBackDataFromDays) on next pull request

Reply:
Thanks for your work

Reply:
You are Welcome